src/components/AreaSelector/AreaSelector.js

```diff
@@ -0,0 +1,24 @@
+import {SelectWithValueReplacement as Select} from 'pcic-react-components'
+import React from 'react'
+
+function AreaSelector({regionNames, onChange, currentRegion}) {
+  var valueOptions = [];
+  regionNames.forEach(name => valueOptions.push({"value": name, "label": name}));
```

Comment:
`_.map` would do this without explicitly constructing and pushing onto an initially empty array. Shorter, fewer oppos for mistakes.